### PR TITLE
Update readme to recommend CLI approach

### PR DESCRIPTION
**Pull request by wlonkly**
Looks like Neutral Face is now unmaintained, and Slack's rate limiting makes it really frustrating to use. Instead, let's recommend a commandline tool that supports rate limits.

Also corrects error_bad_name_i18n explanation.

Also also the plural of emoji is emoji, just like the plural of sushi is sushi.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,33 +2,65 @@
...
+
+If you're comfortable in a terminal, this is the recommended approach, as it
+handles Slack's rate limit.
+
+1. Install [slack-emoji-upload](https://github.com/sgreben/slack-emoji-upload)
+1. Get an `xoxs-*` Slack token following
+   [these instructions](https://github.com/jackellenberger/emojme#finding-a-slack-token).
+   (The team/email/password approach has never worked for me, but token works great.)
+1. Stick the token in a variable, to keep it out of your shell history:
+```
```
[the remark this answers is not in the document]
Comment (wlonkly) [reply README.md]: TIL, thanks, change incoming.

**Comment by SeanPrashad** [comment]
Also I think we may need to rebase off master since we landed #6 recently

**Comment by wlonkly** [comment]
All set! Jeez, I wish Github would combine all those suggestion commits into one. Let me know if you want me to rebase this before you merge.

> RE: Status of Neutral Face Emoji Tools - I was actually thinking of creating a version of Neutral face emoji since the license allows us to (granted we maintain the ISC license), land the patch that respects Slack's rate-limiting policy and publish it on the Chrome store!

Heh, I thought about this earlier, fought with `npm` for about 30 minutes, and went "meh". But I'm no JS developer. 😄 I wonder if Chrome has... _concerns_ about uploading copies of other extensions?

If nothing else you could provide an unpacked but built extension people could install.

File: README.md [hunk 2]
```diff
@@ -2,39 +2,65 @@
...
-## Uploading Emojis
-
-1. Ensure you are logged into the correct Slack workspace
 1. Clone or [download this
    repository](https://github.com/SeanPrashad/slackmojis/archive/master.zip) to
    your local machine. If you have downloaded the `.zip` file, unzip it using a
    third-party or built-in tool made for extracting archives.
+1. Follow one of the options under [Uploading Options](#Uploading-Options) below.
+
+### Uploading Options
```
Comment (SeanPrashad) [inline README.md]:
```suggestion
   ### Uploading Options

   #### Option 1: `slack-emoji-upload` command

   If you're comfortable in a terminal, this is the recommended approach, as it
   handles Slack's rate limit.

   1. Install [slack-emoji-upload](https://github.com/sgreben/slack-emoji-upload)
   1. Get an `xoxs-*` Slack token following
      [these instructions](https://github.com/jackellenberger/emojme#finding-a-slack-token).
      (The team/email/password approach has never worked for me, but token works great.)
   1. Stick the token in a variable, to keep it out of your shell history:
      ```
      $ read -s TOKEN
      [paste token and hit enter]
      ```
   1. Change to the directory you want to import emoji from
   1. Import them like so, substituting the name of your slack workspace. The `xargs` works
      around an open-files bug I encountered, and the rate-limit (one every 4s or 15/minute)
      is just under Slack's reported 20-request-per-minute limit.
      ```
      $ ls -1 | xargs -n 20 slack-emoji-upload -team YOUR_SLACK_TEAM -token $TOKEN -rate-limit 4s
      ```

   #### Option 2: Chrome extension

   This is browser-based, but doesn't respect Slack's rate limit. Unfortunately it looks
   like the author of the extension has stopped working on it, as there are rate-limiting
   patches [available](https://github.com/Fauntleroy/neutral-face-emoji-tools/pull/15) for
   months but unmerged.

   1. Install [Neutral Face
      Emoji Tools](https://chrome.google.com/webstore/detail/neutral-face-emoji-tools/anchoacphlfbdomdlomnbbfhcmcdmjej)
   1. From within your Slack workspace, click on the Workspace name and
      click `Customize Slack`. Your web browser should open a new tab directly to
      the `Customize Your Workspace` page.
   1. Open finder or windows explorer, select up to 100 emoji and then drag and drop
      them into the Bulk Emoji Uploader section.
   1. Wait a few minutes before uploading more emoji as Slack's
      rate-limiter will kick in.
```

**Comment by wlonkly** [comment]
Something went horribly wrong with that Suggested Change. Fixing.

**Pull request merged**